Makefile.in

```diff
@@ -133,7 +138,7 @@ unittest/run$(EXEEXT): $(base_objs) $(test_objs) $(extra_libs)
 unittest/main.o: unittest/suites.h
 
 unittest/suites.h: $(test_suites) Makefile
-	sed -n 's/TEST_SUITE(\(.*\))/SUITE(\1)/p' $(test_suites) >$@
+	sed -n 's/TEST_SUITE(\(.*\))/SUITE(\1)/p' $^ | grep -v sed >$@
```

Comment:
A developer who adds a test suite with "sed" in the name won't be happy with this. :slightly_smiling_face: Also, sed-ing here feels like a workaround for the real problem.

Perhaps change

    test_suites = @test_suites@

to

    test_suites = $(addprefix $(srcdir)/,@test_suites@)

instead?

Reply:
You are of course right, and it was indeed more of a workaround.

Reply:
Do we want to introduce GNU make as a dependency too, or use some portable construct ?

Reply:
> Do we want to introduce GNU make as a dependency too, or use some portable construct ?

Good catch. I think that it would be preferable to continue not requiring GNU make for non-dev-mode.

Reply:
Flipped the grep instead